src/librustdoc/clean/inline.rs

```diff
@@ -600,7 +600,7 @@ fn build_module_items(
                 items.push(clean::Item {
                     name: None,
                     attrs: Box::new(clean::Attributes::default()),
-                    item_id: ItemId::Primitive(prim_ty, did.krate),
+                    item_id: ItemId::DefId(DefId { index: 0u32.into(), krate: did.krate }),
```

Comment:
This change is very ugly though. I wondered if making `item_id` an `Option` would have been better but in the end I just left as is. If you have a better idea, I'm all for it. Otherwise I can add a comment.

Reply:
Wouldn't `u32::MAX` be better?

Reply:
That would work too :)